### PR TITLE
chore: update genai notebooks for Gemini

### PR DESCRIPTION
**Pull request by GarrettWu**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: notebooks/generative_ai/bq_dataframes_llm_code_generation.ipynb [hunk 1]
```diff
@@ -383,15 +396,28 @@
...
+      "execution_count": 9,
       "metadata": {
         "id": "sdjeXFwcHfl7"
       },
-      "outputs": [],
+      "outputs": [
+        {
+          "data": {
+            "text/html": [
+              "Query job a3897125-4272-4817-a0e6-8e1a9e022b93 is DONE. 0 Bytes processed. <a target=\"_blank\" href=\"https://console.cloud.google.com/bigquery?project=bigframes-dev&j=bq:US:a3897125-4272-4817-a0e6-8e1a9e022b93&page=queryresults\">Open Job</a>"
```
Comment (milkshakeiii) [inline notebooks/generative_ai/bq_dataframes_llm_code_generation.ipynb:407]: I believe this may have been added accidentally? (need to clear outputs)

**Review by milkshakeiii — APPROVED** [review]
Might need to clear outputs. Other than that LGTM.

File: notebooks/generative_ai/bq_dataframes_llm_code_generation.ipynb (hunk 1 above)
Comment (GarrettWu) [reply notebooks/generative_ai/bq_dataframes_llm_code_generation.ipynb:407]: Is it a requirement somewhere that we shouldn't include outputs? In our code base it is in consistent that some contains outputs, but others not.

Comment (milkshakeiii) [reply notebooks/generative_ai/bq_dataframes_llm_code_generation.ipynb:407]: Hm, I guess it's probably harmless or potentially helpful to include them. I just thought it seemed cleaner without them.

**Pull request merged**